File: src/main/java/superstitio/cardModifier/modifiers/block/MilkBlock.java
package superstitio.cardModifier.modifiers.block;

import com.badlogic.gdx.graphics.Color;
import com.evacipated.cardcrawl.mod.stslib.blockmods.AbstractBlockModifier;
import com.megacrit.cardcrawl.cards.DamageInfo;
import superstitio.DataManager;
import superstitio.cardModifier.modifiers.AbstractLupaBlock;

public class MilkBlock extends AbstractLupaBlock {

    public static final String ID = DataManager.MakeTextID(MilkBlock.class);

    public MilkBlock() {
        super(ID);
    }

    @Override
    public void onThisBlockDamaged(DamageInfo info, int lostAmount) {

    }

    @Override
    public AbstractBlockModifier makeCopy() {
        return new MilkBlock();
    }

    @Override
    public Color blockImageColor() {
        return Color.WHITE.cpy();
    }
}

File: superstitioMain/java/superstitio/cardModifier/modifiers/block/MilkBlock.kt
package superstitio.cardModifier.modifiers.block

import com.badlogic.gdx.graphics.Color
import com.evacipated.cardcrawl.mod.stslib.blockmods.AbstractBlockModifier
import com.megacrit.cardcrawl.cards.DamageInfo
import superstitio.DataManager
import superstitio.cardModifier.modifiers.AbstractLupaBlock

class MilkBlock : AbstractLupaBlock(ID)
{
    override fun onThisBlockDamaged(info: DamageInfo, lostAmount: Int)
    {
    }

    override fun makeCopy(): AbstractBlockModifier
    {
        return MilkBlock()
    }

    override fun blockImageColor(): Color
    {
        return Color.WHITE.cpy()
    }

    companion object
    {
        val ID: String = DataManager.MakeTextID(MilkBlock::class.java)
    }
}